### ISSUE TITLE
Custom Attributes do not work with <compose> rendered custom elements

### ISSUE THREAD

**Issue by st0n3-man** [aurelia/templating#486]
https://gist.run/?id=331b55cbbea0458f98d4a1104c0ba242

My custom attribute does not work for compose containerless rendered elements.
In above gist, first and third work, second doesn't.

Since my custom elements are going to be svg based, as per aurelia docs I have to make them containerless.

**Comment by EisenbergEffect** [comment]
What element should it apply to? A containerless element literally has no host element so you can't apply an attribute to it.

**Comment by st0n3-man** [comment]
I understand. 
Maybe I should rephrase this as a query.
Containerless composed custom elements do not apply the custom attributes. Is there anything I could do as a work-around? Like cascading down the custom attributes to the template of custom element. 
I hope I am making sense.
